scripts/exglobal_fcst_nemsfv3gfs.sh

```diff
@@ -697,7 +697,8 @@ wav_petlist_bounds=" $((NTASKS_FV3)) $((NTASKS_FV3m1+npe_wav))"
 cat > nems.configure <<EOF
 EARTH_component_list: ATM WAV
 EARTH_attributes::
-  Verbosity = 0
+  Verbosity = high
```

Comment:
Do "Verbosity" control the about to run time output?  If so, do we need high now that waves are working correctly?   The GSI has a "verbose" flag.  We set it to .true. for development.  We set it to .false. in parallels to reduce the size of the log file and reduce job run time.

Reply:
@yangfanglin this was a change recommended by Gerhard. Pls comment on what should be done.

Reply:
please change it to low (or no?).  We do not need it in mass production.